resources/clients/react/elwing/README.md

```diff
@@ -0,0 +1,83 @@
+# Developer guide
+
+Elwing is a bootstrapped companion app to the Cloudscape design system. Its opinionated
+layout accelerates development time by exposing a simple plugin API allowing a developer
+to focus on defining pages and content rather than app structure.
```

Comment:
Update last sentence: 
Its opinionated layout accelerates development time by exposing a simple plugin API. Developers
can use the plugin API to focus on defining pages and content instead of app structure.